.github/workflows/docs.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         with:
           python-version: 3.7
 
+      - run: sudo npm install katex@0.12.0 -g
```

Comment:
pin the version here as in the `layout.html`

Reply:
do you know if there is a special tag like latest to install the latest version here and in the layout ?

Reply:
TIL; you can use `latest` in the tag, didnt know that.
https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/
https://cdn.jsdelivr.net/npm/katex@latest/dist/

Same size

So, do you want to use latest version both in CI and layout.html?

Reply:
yes, please

Reply:
Done